generate/templates/couchbase-server/Dockerfile.template

```diff
@@ -13,11 +46,10 @@ LABEL maintainer="docker@couchbase.com"
 #  net-tools: ifconfig, arp, netstat
 #  numactl: numactl
 RUN set -x && \
-    apt-get update && \
-    apt-get install -yq runit wget tzdata \
+    {{ .PKG_COMMAND }} update -y -q && \
+    {{ .PKG_COMMAND }} install -y -q runit wget tzdata \
```

Comment:
Icky .. this only "works" because yum has the terrible misfeature that it won't fail with an error if some of the packages you try to install don't exist (like runit). Do you think you could move "runit" to .CB_EXTRA_DEPS and have that be dependent on the specific platform being built? It's a minor issue now, but if we wanted to integrate the current Red Hat Dockerfile into this mechanism (which would be a great idea!), it could be more problematic - especially if there were any dependency packages that were unique to the yum side of things.

Reply:
moved to CB_EXTRA_DEPS.